packages/gatsby-plugin-mdx/utils/gen-mdx.js

```diff
@@ -1,14 +1,10 @@
-const babel = require(`@babel/core`)
+const sucrase = require(`sucrase`)
 const grayMatter = require(`gray-matter`)
 const mdx = require(`@mdx-js/mdx`)
-const objRestSpread = require(`@babel/plugin-proposal-object-rest-spread`)
 
 const debug = require(`debug`)(`gatsby-plugin-mdx:gen-mdx`)
 
 const getSourcePluginsAsRemarkPlugins = require(`./get-source-plugins-as-remark-plugins`)
-const htmlAttrToJSXAttr = require(`./babel-plugin-html-attr-to-jsx-attr`)
-const removeExportKeywords = require(`./babel-plugin-remove-export-keywords`)
-const BabelPluginPluckImports = require(`./babel-plugin-pluck-imports`)
```

Comment:
These plugins can also be dropped from the monorepo but I didn't do that in this PR.